packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5560,6 +5578,11 @@ class RawImage extends LeafRenderObjectWidget {
       ..filterQuality = filterQuality;
   }
 
+  @override
+  void didUnmountRenderObject(RenderImage renderObject) {
+    renderObject.image = null;
```

Comment:
This doesn't need to be disposed?

Reply:
This causes the render object to dispose the image.

Reply:
I've refactored `RawImage` into a public `StatefulWidget` and made the current impl private. The public impl now does not need to be special cased.